app.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ShapeShift",
-  "version": "3.3.1",
+  "version": "3.4.0",
```

Comment:
Avoid pushing OTA for this one so we ensure users have the expo-application native files